### DISCUSSION TITLE
The count of thread is high?  uvicorn+fastapi

### DISCUSSION THREAD

**Discussion by w0shlzheNdecaI** [fastapi/fastapi#8432] [Questions]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
...
```

### Description

I use uvicorn +fastapi,  the cmd is "uvicorn main:app --host 0.0.0.0 --reload --port 80"
I spent a day observing my application, when I use ps -xH ，the thread of python always increase, such as 15:00 the count is 19, ,18:00 the count is 52 , but the next day morning the count is more than 100.  And "VIRT" in  cmd of "top" more than 10g but the "RES" just 200M

### Operating System

Linux

### FastAPI Version

0.88.0

### Python Version

3.10

**Comment by iudeen** [comment]
Is it the same without `--reload`? 

It's really hard to answer your question without having more information on what your application does (some kind of MRE).  It'd be great if you can provide us with an MRE.
